src/measure_theory/integral/set_integral.lean

```diff
@@ -141,6 +141,28 @@ begin
   ... = ∫ x in s, f x ∂μ : by simp
 end
 
+lemma of_real_set_integral_one_of_measure_ne_top {α : Type*} {m : measurable_space α}
+  {μ : measure α} {s : set α} (hs : μ s ≠ ∞) :
+  ennreal.of_real (∫ x in s, (1 : ℝ) ∂μ) = μ s :=
+calc
+ennreal.of_real (∫ x in s, (1 : ℝ) ∂μ)
+= ennreal.of_real (∫ x in s, ∥(1 : ℝ)∥ ∂μ) : by simp only [cstar_ring.norm_one]
+... = ∫⁻ x in s, 1 ∂μ :
+begin
+  rw of_real_integral_norm_eq_lintegral_nnnorm,
+  { simp only [nnnorm_one, ennreal.coe_one] },
+  { rw integrable_const_iff,
+    simp only [hs.lt_top, one_ne_zero, measure.restrict_apply, measurable_set.univ, set.univ_inter,
+      false_or], }
+end
+... = μ s :
+by simp only [lintegral_one, measure.restrict_apply, measurable_set.univ, set.univ_inter]
```

Comment:
```suggestion
... = μ s : set_lintegral_one _
```
I was about to suggest that this last part ought to be its own lemma, but it already is!